mocks/service/discord_service_mock.go

```diff
@@ -0,0 +1,80 @@
+// Code generated by mockery v2.43.2. DO NOT EDIT.
```

Comment:
If this file are generated, can you consider these:
- Use [go generate](https://go.dev/blog/generate) to specifically tell which file are requiring this file and the command to generate this file.
- Add a step in the workflow to generate this file and ensuring the one committed is already generated correctly. E.g.:
  ```
  - name: Generate
    run: |
      go generate -v ./...
      git diff --exit-code HEAD
  ```
- Add a `.gitattribute` that specify if this file is generated and the diff should be ignored. E.g.:
  ```
  *_mock.go -diff linguist-generated
  ```

Reply:
all the generations are handled by `mockery` and configured inside the `.mockery.yaml`